vignettes/retrieved_dropout.Rmd

```diff
@@ -0,0 +1,318 @@
+- Study drug discontinuation is simulated to have no effect on the mean trajectory in the placebo group.
+In the intervention group, subjects who discontinue follow the slope of the mean trajectory from the placebo group
+from that time point onward. This is compatible with a copy increments in reference (CIR) assumption. 
+- Study dropout at the study drug discontinuation visit occurs with a probability of 50% leading to missing outcome
+data from that time point onward.
+
+The function `simulate_data()` requires 3 arguments (see the function documentation `help(simulate_data)` for more details): 
+
+- `pars_c`: The simulation parameters of the control group.
+- `pars_t`: The simulation parameters of the intervention group.
+- `post_ice1_traj`: Specifies how observed outcomes after ICE1 are simulated.
+
+
+Below, we report how data according to the specifications above can be simulated with function `simulate_data()`:
+
+```{r}
+library(rbmi)
+library(dplyr)
+
+set.seed(1392)
+
+n <- 100
+time <- c(0, 3, 6, 9, 12)
+
+# Mean trajectory control
+muC <- c(50.0, 52.5, 55.0, 57.5, 60.0)
+
+# Mean trajectory intervention
+muT <- c(50.0, 52.5, 55.0, 56.25, 57.50)
+
+# Create Sigma
+sd_error <- 2.5
+covRE <- rbind(
+  c(25.0, 6.25),
+  c(6.25, 25.0)
+)
+
+Sigma <- cbind(1, time / 12) %*% covRE %*% rbind(1, time / 12) + diag(sd_error^2, nrow = length(time))
+
+# Set probability of discontinuation
+probDisc_C <- 0.03
+probDisc_T <- 0.04
+or_outcome <- 1.10 # +1 point increase => +10% odds of discontinuation
+
+# Set dropout rate following discontinuation
+prob_dropout <- 0.5
+
+# Set simulation parameters of the control group
+parsC <- set_simul_pars(
+    mu = muC,
+    sigma = Sigma,
+    n = n,
+    prob_ice1 = probDisc_C,
+    or_outcome_ice1 = or_outcome,
+    prob_post_ice1_dropout = prob_dropout
+)
+
+# Set simulation parameters of the intervention group
+parsT <- parsC
+parsT$mu <- muT
+parsT$prob_ice1 <- probDisc_T
+
+# Set assumption about post-ice trajectory
+post_ice_traj <- "CIR"
+
+# Simulate data
+data <- simulate_data(
+    pars_c = parsC,
+    pars_t = parsT,
+    post_ice1_traj = post_ice_traj
+) %>%
+  select(-c(outcome_noICE, ind_ice2)) # remove unncessary columns
+  
+
+head(data)
+```
+
+The frequency of the ICE and proportion of data collected after the ICE impacts the variance of the treatment effect for 
+retrieved dropout models. For example, a large proportion of ICE combined with a small proportion of data collected after the 
+ICE might result in substantial variance inflation, especially for more complex retrieved dropout models.
+
+The proportion of subjects with an ICE and the proportion of subjects who withdrew from the simulated study is summarized below:
+
+```{r}
+
+# Compute endpoint of interest: change from baseline
+data <- data %>% 
+  filter(visit != "0") %>%
+  mutate(
+    change = outcome - outcome_bl,
+    visit = factor(visit, levels = unique(visit))
+  )
+      
+
+data %>% group_by(visit) %>% summarise(
```

Comment:
The 3% discontinutation prob is for a subject with mean equal to the baseline mean (i.e. 50). Subjects with a higher outcome (i.e. more severe) have a higher probability of discontinuing. Since the mean increases over time, on average there are more discontinuations at later visits.